data-prepper-plugins/mapdb-prepper-state/src/test/java/com/amazon/dataprepper/plugins/prepper/state/MapDbPrepperStateTest.java

```diff
@@ -62,6 +62,9 @@ public String apply(byte[] bytes, DataClass s) {
                 data4.stringVal
         )));
 
+        Assert.assertTrue(prepperState.sizeInBytes() > 0);
```

Comment:
Is this the best we can do here? It should be clear what the size is right? I thought a previous revision had an exact size

Reply:
I changed this in case the size is varied. I can revert this.

Reply:
Well if the size varies, we should just verify that its in the range in which is varies. > 0 is probably not restrictive enough to catch issues if they arise.

Reply:
Makes sense, will do